js/index.js

```diff
@@ -0,0 +1,46 @@
+$(function(){
+		// On click run function changeCity
+		$("#submit-btn").click(changeCity);
+		
+		// Set variable
+		var city;
+		
+		// Set function - notice (event) in the function applies 
+		// to the prevent default
+		function changeCity(event){
+		// Apply preventDefault within function
+		event.preventDefault();
+
+		// Pull user information from the text-field 
+		// Use ".toLowerCase()" to make site more user friendly
+			city = $("#city-type").val().toLowerCase();
```

Comment:
:+1: Nice use of `toLowerCase` here :) This makes it more versatile to accept a users input.